src/main/java/pl/jalokim/propertiestojson/object/ObjectJsonType.java

```diff
@@ -19,18 +16,18 @@
     private Map<String, AbstractJsonType> fields = new HashMap<>();
 
     public void addField(final String field, final AbstractJsonType object, PathMetadata currentPathMetaData) {
-        if(object instanceof SkipJsonField) {
+        if (object instanceof SkipJsonField) {
             return;
         }
 
         AbstractJsonType oldFieldValue = fields.get(field);
-        if(oldFieldValue != null) {
-            if(oldFieldValue instanceof MergableObject && object instanceof MergableObject) {
+        if (oldFieldValue != null) {
+            if (oldFieldValue instanceof MergableObject && object instanceof MergableObject) {
                 mergeObjectIfPossible(oldFieldValue, object, currentPathMetaData);
             } else {
                 throw new CannotOverrideFieldException(currentPathMetaData.getCurrentFullPath(),
-                                                       oldFieldValue,
```

Comment:
this should stay like earlier, 
or please move first arugment to next line, and every of arugments should be in the same line.